### PR TITLE
Less integration

### PR DESCRIPTION
**Pull request by eimfach**

### CODE REVIEW

File: build/webpack.config.js [hunk 1]
```diff
@@ -142,6 +142,17 @@ webpackConfig.module.loaders.push({
 })
 
 webpackConfig.module.loaders.push({
+  test: /\.less/,
```
Comment (layflags) [inline build/webpack.config.js]: What impact does it have, when we switch from `/\.css$/` to `/\.less/`?

Comment (layflags) [inline build/webpack.config.js]: Aaah, we test for both, ok!

Comment (layflags) [inline build/webpack.config.js]: Does it work that way?

Comment (eimfach) [inline build/webpack.config.js]: it think this is for supporting css 'modules' in .css files (if you use them), but not sure.
At this state we have the option to use .css (with fancy loaders, modules eg. ), .less and .sass

btw: the $ delimiter was missing

**Comment by layflags** [comment]
looks good so far :+1:

**Pull request merged**